resources/todomvc/architecture-examples/preact/.babelrc

```diff
@@ -1,9 +1,9 @@
 {
-  "presets": [
-    "es2015-loose",
-    "stage-0"
-  ],
-  "plugins": [
-    ["transform-react-jsx", { "pragma":"h" }]
-  ]
+    "presets": [
+        "@babel/env",
+        "@babel/preset-react"
+    ],
+    "plugins": [
+        "@babel/plugin-proposal-class-properties"
+    ]
 }
```

Comment:
Do we want an explicit browserslist query?

Reply:
we actually don't need the .babelrc file - will remove